### PR TITLE
Added localisation support part 2

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Added localisation support

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/utils/BotsUtils.ts [hunk 1]
```diff
@@ -11,16 +11,14 @@
  *  limitations under the License.
  */
 
+import { t } from 'i18next';
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/BotsUtils.ts:14]: I would recommend using hook if we are using this in functional components.

File: openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx [hunk 2]
```diff
@@ -52,12 +52,10 @@ import SVGIcons, { Icons } from './SvgUtils';
 
 export const getBadgeName = (tableType?: string) => {
   switch (tableType) {
-    case 'REGULAR':
-      return 'table';
     case 'QUERY':
-      return 'query';
+      return toLower(i18n.t('label.query'));
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx:56]: For consistency, we should directly import `t` from the `i18next` and user here.

File: openmetadata-ui/src/main/resources/ui/src/utils/BotsUtils.ts (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/utils/BotsUtils.ts:14]: this is util file!

File: openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx:56]: sure!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**